Packs/Kafka/Integrations/KafkaV3/KafkaV3.py

```diff
@@ -801,18 +901,23 @@ def main():  # pragma: no cover
 
     # Should we use SSL
     use_ssl = demisto_params.get('use_ssl', False)
-
-    if use_ssl:
-        # Add Certificates
-        ca_cert = demisto_params.get('ca_cert', None)
-        client_cert = demisto_params.get('client_cert', None)
-        client_cert_key = demisto_params.get('client_cert_key', None)
-        ssl_password = demisto_params.get('additional_password', None)
-        kafka_kwargs = {'brokers': brokers, 'ca_cert': ca_cert, 'client_cert': client_cert,
-                        'client_cert_key': client_cert_key, 'ssl_password': ssl_password, 'offset': offset,
-                        'trust_any_cert': trust_any_cert, 'group_id': group_id}
-    else:
-        kafka_kwargs = {'brokers': brokers, 'offset': offset, 'trust_any_cert': trust_any_cert, 'group_id': group_id}
+    # Should we use SASL (with SSL and PLAIN)
+    use_sasl = demisto_params.get('use_sasl', False)
+
+    ca_cert = demisto_params.get('ca_cert', None)
+    client_cert = demisto_params.get('client_cert', None)
+    client_cert_key = demisto_params.get('client_cert_key', None)
+    ssl_password = demisto_params.get('additional_password', None)
+    plain_username = demisto_params.get('credentials', {}).get('identifier')
+    plain_password = demisto_params.get('credentials', {}).get('password')
+
+    kafka_kwargs = {'use_ssl': use_ssl, 'brokers': brokers, 'ca_cert': ca_cert, 'offset': offset,
+                    'use_sasl': use_sasl, 'group_id': group_id,
+                    'trust_any_cert': trust_any_cert,
+                    'client_cert': client_cert, 'client_cert_key': client_cert_key,
+                    'plain_username': plain_username, 'plain_password': plain_password}
+    if ssl_password:
```

Comment:
Why was this needed?

Reply:
It is not supposed to be sent if it is None